File: 1929-concatenation-of-array/1929-concatenation-of-array.py
class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        res = [None]*(2*len(nums))
        
        i = 0
        
        while i < len(nums):
            res[i],res[i+len(nums)] = nums[i],nums[i]
            i+=1
            
        return res

File: 1929-concatenation-of-array/1929-concatenation-of-array.cpp
class Solution {
public:
    vector<int> getConcatenation(vector<int>& nums) {
        vector<int> res(2*nums.size(),0);
        
        for(int i=0;i<nums.size();i++){
            res[i] = nums[i];
            res[i+nums.size()] = nums[i];
        }
        return res;
    }
};